packages/@aws-cdk/aws-ec2/test/cfn-init-element.test.ts

```diff
@@ -206,6 +206,12 @@ describe('InitFile', () => {
     });
   });
 
+  test('empty content string throws error', () => {
+    expect(() => {
+      ec2.InitFile.symlink('/tmp/foo', '');
```

Comment:
Why have you changed `fromString` and are you  testing `symlink` ?

Reply:
Typo, fixed.